praw/models/reddit/submission.py

```diff
@@ -522,6 +522,33 @@ def nsfw(self):
             API_PATH["marknsfw"], data={"id": self.thing.fullname}
         )
 
+    def set_original_content(self):
+        """Mark as original content.
+
+        This method can only be used by moderators of the subreddit
+        that the submission belongs to, if the subreddit has enabled
+        the beta feature in settings.
+
+        Example usage:
+
+        .. code:: python
+
+           submission = reddit.subreddit('test').submit('oc test',
+                                                        selftext='original')
+           submission.mod.set_original_content()
+
+        See also :meth:`~.unset_original_content`
```

Comment:
This can be

    :meth:`.unset_original_content`

(without the tilde). The purpose of the tilde is that it lets you specify which class a method belongs to without outputting the class name in the final documentation. For instance, writing

    :meth:`.SubmissionModeration.unset_original_content`

gives you output that looks like `SubmissionModeration.unset_original_content`, but writing

    :meth:`~.SubmissionModeration.unset_original_content`

gives you `unset_original_content`. The tilde is used when you need to specify the class that the method belongs to (which happens when different classes have methods with the same name), but you don't want to print the class name. Since you don't specify a class name, the tilde is unnecessary.

Reply:
Awesome, thank you very much for your clear explanation!